product_packaging_dimension/migrations/14.0.1.0.0/pre-migrate.py

```diff
@@ -5,18 +5,9 @@
 
 
 def migrate(cr, version):
-    # Rename lnght into packaging_length
-    if column_exists(cr, "product_packaging", "lnght"):
-        rename_column(cr, "product_packaging", "lnght", "packaging_length")
 
-        # Convert old hard-coded uom values (mm)
-        # into new default uom values (m)
-        cr.execute(
-            """
-        UPDATE product_packaging
-        SET
-        packaging_length = packaging_length/1000,
-        height = height/1000,
-        width = width/1000,
-        """
-        )
+    # Rename lngth into packaging_length
```

Comment:
This script won't run on existing installations.
You should add a script for `14.0.2.0.0` (I'd go for major as this can be a breaking change).

Reply:
what about this?

Reply:
I don't want any script ran on existing installation only migrated data 13->14
if the current version of the addon is installed, harm is already done, 
about the complexity of handling the current data, I pointed here 
https://github.com/OCA/product-attribute/pull/957#discussion_r767633335